gui.js

```diff
@@ -329,6 +329,26 @@ IDE_Morph.prototype.openIn = function (world) {
     */
     function interpretUrlAnchors() {
         var dict;
+	function doDictFlags() {
+            if (dict.editMode) {
+                myself.toggleAppMode(false);
+            } else {
+                myself.toggleAppMode(true);
+            }
+
+            if (!dict.noRun) {
+                myself.runScripts();
+            }
+
+            if (dict.hideControls) {
+                myself.controlBar.hide();
+                window.onbeforeunload = nop;
```

Comment:
I just cut-and-pasted this code from the #present branch into a separate procedure so I could use it in two places.  I didn't change any details.